.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: java
```

Comment:
You can specify you don't want to use a specific language. Use `language: minimal` :smiley: